app/lib/service/security_advisories/backend.dart

```diff
@@ -212,6 +212,28 @@ List<String> sanityCheckOSV(OSV osv) {
     errors.add('Invalid id, id too long (over 255 characters).');
   }
 
+  if (osv.affected == null || osv.affected!.isEmpty) {
```

Comment:
hmm, I wonder if we should add annotations such that `affected` has a default value empty-list:

https://pub.dev/documentation/json_annotation/4.8.1/json_annotation/JsonKey/defaultValue.html

Same for `affected.versions` it would make life simpler.

------

If we try this, let's do so in a separate PR.

Reply:
Noted.